cmd/config.go

```diff
@@ -38,39 +40,29 @@ const (
 	minioConfigBackupFile = minioConfigFile + ".backup"
 )
 
-func saveServerConfig(ctx context.Context, objAPI ObjectLayer, config *serverConfig) error {
+func saveServerConfigIndent(ctx context.Context, objAPI ObjectLayer, config *serverConfig) error {
 	if err := quick.CheckData(config); err != nil {
 		return err
 	}
 
-	data, err := json.MarshalIndent(config, "", "\t")
+	configBytes, err := json.MarshalIndent(config, "", "\t")
```

Comment:
We don't need to indent and save, `mc admin config get` indents and returns the values. On-disk value can be smaller.